packages/bitcoin/src/rpc.ts

```diff
@@ -80,11 +80,19 @@ export const makeRequest = async <T extends keyof RpcRequest & keyof typeof resp
     }),
   });
 
-  if (res.status !== 200) {
```

Comment:
do we need to adjust this for our 25324 other rpc clients too?

Reply:
lol no this seems to be something particular to the bitcoin rpc node